custom-connectors/EleadSalesOpportunities/apiProperties.json

```diff
@@ -0,0 +1,8 @@
+{
+  "properties": {
+    "connectionParameters": {},
+    "iconBrandColor": "#1F1A3C",
+    "capabilities": [],
+    "publisher": "Bhargava, Aman Avinash"
```

Comment:
As with the others, please add the `stackOwner` property, and remove PII in favor of using company-specific information for both stackOwner and publisher.